### PR TITLE
v0.11.2 manual migrate to Rust

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@Anthchirp @blabalme @conda-forge/core Previous auto PRs filed with no action for a couple months, I'm trying to manually migrate this to the new Rust implementation upstream. Requesting review and merge. Thanks!

**Comment by xhochy** [comment]
Please leave a window of 3-5 days between the maintainer and the conda-forge/core ping. If @conda-forge/jellyfish doesn't react in that timeframe; I'm happy to merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,16 +14,19 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 2
+# Also skip for ARM64 due to Maturin no support for now, and when ppc64le AND python<3.10
```
Comment (xhochy) [inline recipe/meta.yaml:17]: > ARM64 due to Maturin no support for now,

`maturin` should build just fine on `osx-arm64`.

Comment (thewchan) [reply recipe/meta.yaml:17]: Maturin did complain about "bad CPU type" when building for ARM64? Not sure why then?

Comment (xhochy) [reply recipe/meta.yaml:17]: You also need to put `maturin` into build using the `# [build_platform != target_platform]` selector.

Comment (thewchan) [reply recipe/meta.yaml:17]: Sorry @xhochy for falling off; picking this up again. I just added maturin to build and selector as well.

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@xhochy @conda-forge/core okay I think this time it's really ready for merging!

**Review by xhochy — APPROVED** [review]

**Pull request merged**